### PR TITLE
Use host for cmake and compiler('cxx'). Stop pinning blas.

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
cc @conda-forge/core I am not pinning openblas anymore, and relying on the conda-forge pinning instead. I hope that this is the way to go.

**Comment by isuruf** [comment]
You should remove the pinning in run as well.

**Comment by SylvainCorlay** [comment]
Thank you, that was an oversight.

File: .ci_support/win_cxx_compilervs2015vc14.yaml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 cxx_compiler:
 - vs2015
-openblas:
-- 0.2.20
```
Comment (ocefpaf) [inline .ci_support/win_cxx_compilervs2015vc14.yaml]: This should stay, no? My guess is that re-rendering will re-add those.

Comment (isuruf) [reply .ci_support/win_cxx_compilervs2015vc14.yaml]: `openblas` is not used in windows

Comment (ocefpaf) [reply .ci_support/win_cxx_compilervs2015vc14.yaml]: Ah OK. But we do have a working version of openblas on Windows though, right?

**Comment by isuruf** [comment]
Is this is header only library? If so, there's no openblas pinning needed. You can have `- openblas >= 0.2.20` in both run and host.

File: .ci_support/win_cxx_compilervs2015vc14.yaml (hunk 1 above)
Comment (SylvainCorlay) [reply .ci_support/win_cxx_compilervs2015vc14.yaml]: We have a fallback when no blas version is available. openblas used to not be available on windows.

I will open a separate PR for this.

**Comment by SylvainCorlay** [comment]
> Is this is header only library? If so, there's no openblas pinning needed. You can have - openblas >= 0.2.20 in both run and host.

Yes, it is header only. I would like to include the tests when modern compilers kick in.

**Comment by SylvainCorlay** [comment]
> Btw, is it intentional that files like $PREFIX/include/xflens/cxxlapack/README.md are included? Looks weird to me.

Not really. But it is more of an issue with the way `xtensor-blas` packages the content of the xflens directory. (We vendor a maintained fork of the unmaintained flex, internally called xflens. We should add an exclude pattern in our cmake).

Opening an issue.

**Pull request merged**